### PR TITLE
Fix typo in basic-css-hex-code-colors. Add article -the- to description.

### LINKED ISSUE #13581: Typo in "HTML5 and CSS: Use Hex Code for Specific Colors"
https://www.freecodecamp.com/challenges/use-hex-code-for-specific-colors

"Like decimals, the symbols 0-9 **represents** values zero to nine."

It should be "...symbols 0-9 **represent** values...", right?

Otherwise, congratulations for the tremendous work on the site.
This is very helpful.

### PR DESCRIPTION
**Pull request by homerrowsky**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13581 

#### Description

- This change is a **first-timers-only** contribution. 
- **Adds article _'the'_ to a sentence** in the challenge's description.
- This is an additional change suggested by @erictleung after one made by @eti8.

closes #13581

### CODE REVIEW

**Review by ooade — APPROVED** [review]

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 1]
```diff
@@ -3121,7 +3121,7 @@
...
-        "We usually use <code>decimals</code>, or base 10 numbers, which use the symbols 0 to 9 for each digit. <code>Hexadecimals</code> (or <code>hex</code>) are base 16 numbers. This means it uses sixteen distinct symbols. Like decimals, the symbols 0-9 represent values zero to nine. Then A,B,C,D,E,F represent values ten to fifteen. Altogether, 0 to F can represent a digit in <code>hexadecimal</code>, giving us 16 total possible values. You can find more information about <a target='_blank' href='https://en.wikipedia.org/wiki/Hexadecimal'>hexadecimal numbers here</a>.",
+        "We usually use <code>decimals</code>, or base 10 numbers, which use the symbols 0 to 9 for each digit. <code>Hexadecimals</code> (or <code>hex</code>) are base 16 numbers. This means it uses sixteen distinct symbols. Like decimals, the symbols 0-9 represent values zero to nine. Then A,B,C,D,E,F represent the values ten to fifteen. Altogether, 0 to F can represent a digit in <code>hexadecimal</code>, giving us 16 total possible values. You can find more information about <a target='_blank' href='https://en.wikipedia.org/wiki/Hexadecimal'>hexadecimal numbers here</a>.",
```
Comment (erictleung) [inline seed/challenges/01-responsive-web-design/basic-css.json]: I think [we agreed](https://github.com/freeCodeCamp/freeCodeCamp/issues/13581#issuecomment-282499672) to add in "the" in this sentence as well.

```diff
- Like decimals, the symbols 0-9 represent values zero to nine. 
+ Like decimals, the symbols 0-9 represent the values zero to nine.
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Looks good. Just one more small change. Let us know if you need any help making this change.

File: seed/challenges/01-responsive-web-design/basic-css.json (hunk 1 above)
Comment (homerrowsky) [inline seed/challenges/01-responsive-web-design/basic-css.json]: @erictleung 
**Ouch..**, I passed by the comment of @Greenheart. Sorry :(.
**I suppose I have to make a new pull request, is it right?** (Don't know if there is a _'un-pull request'_)

**Comment by homerrowsky** [comment]
@erictleung :

I did this: 
1. **Modified the file to add Greeenheart's suggestion**
1. **Commited the change** _--(I realized that my question of 'un-pull request' was wrong. I needed to squash commits according to FCC Contributing Guide)_ :open_mouth: 
1. **git pull --rebase upstream staging** _--(Update my local fork of FCC)_ 
1. **git push origin staging** _--(Push the update just done to my remote github fork. OPTIONAL )_
1. **git checkout fix/typo-in-html-and-css** _--(Change to my local branch where edits are made)_
1. **git rebase -i staging** _--(Squash the last two commits of my testing branch, with a rebase against the updated version of my local staging. This was a little painful, never done before)_ :cold_sweat: 
1. **git push -f origin fix/typo-in-html-and-css** _--(Push the squashed commits to my remote branch)_

So that's it. I hope it was well done.
Thanks for your support to all FCC comrades! I've been learning very A LOT from this project.
:+1: :+1: :+1:

**Review by erictleung — APPROVED** [review]
LGTM! 👍  Thanks for your contribution and congrats! 🎉  A small change goes a long way 😄

**Comment by Greenheart** [comment]
@homerrowsky Thanks for contributing! :smile: 🙌

**Comment by QuincyLarson** [comment]
@homerrowsky thanks for your contribution!

**Pull request merged**